config/standard/test/tasks.spec.js

```diff
@@ -678,9 +678,10 @@ function resetDate(object, key, keyToChange, offset) {
 }
 
 function traverse(object, func, key, offset) {
+    // eslint-disable-next-line guard-for-in
```

Comment:
How do you feel about this rule, in general?

I'm not sure from reading why it wouldn't apply here, except that in general the rule isn't that helpful.

Reply:
Maybe I don't love it just because I haven't been bitten by these problems in the past. And perhaps it isn't super relevant for us since we don't modify the prototype or use inheritance in much of our codebase.

> Looping over objects with a for in loop will include properties that are inherited through the prototype chain.

Reply:
In the end, I added json validation during the eslint that runs in travis. That required me to move the plugin to root